lib/plan-builder.js

```diff
@@ -115,114 +118,96 @@ class RdtExpr {
    * Controls features such as the length and type of the generated value.
    * @method planBuilder#maskDeterministic
    * @since 2.7.0
-   * @param { PlanColumnName } [column] - The name of the column to be defined.
-   *  This can be either a string or the return value from PlanBuilder.col(), PlanBuilder.viewCol() or PlanBuilder.schemaCol().
-   * @param { Item } [options] - An object with name-value pairs specific to the redaction method.
```

Comment:
In the JSDoc, the previous documentation of the params is correct both here and in the other public redaction functions:  the caller passes the column and option params.

The ...args syntax is a JavaScript shortcut for setting an array with the first item from the column param and the second item from the options param.